client/src/extension.ts

```diff
@@ -83,12 +83,18 @@ export function activate(context: vscode.ExtensionContext) {
           endpoint,
           new AzureKeyCredential(azureApiKey)
         );
-        //result variable will be used in future implementation
         const result = await client.getChatCompletions(
           deploymentId,
           messages,
           parameters
         );
+        const editor = vscode.window.activeTextEditor;
+        editor.edit((editBuilder) => {

```

Comment:
nit: consider using `vscode.window.activTextEditor` directly

```suggestion
        const editor = vscode.window.activeTextEditor.edit((editBuilder) => {
```